catalog/mgmp/catalog-transformer-mgmp/src/test/resources/schemas/mgmp.xsd

```diff
@@ -0,0 +1,751 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
❓ I notice these artifacts are under test resources. Is it worth including these in `<home>/{schema|schematron}` and maybe configuring some validators for them?